### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

**Comment by AnyoneClown** [comment]
image
image
image
image
image

**Review by pavlejviki — CHANGES_REQUESTED** [review]
add blank lines at the end of your html files

File: taxi/models.py [hunk 1]
```diff
@@ -6,10 +6,16 @@ class Manufacturer(models.Model):
     name = models.CharField(max_length=255, unique=True)
     country = models.CharField(max_length=255)
 
+    def __str__(self):
```
Comment (viktoria-rybenchuk) [inline taxi/models.py]: add type annotation

File: taxi/views.py [hunk 2]
```diff
@@ -13,3 +14,29 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(generic.ListView):
+    model = Manufacturer
+    queryset = Manufacturer.objects.all().order_by("name")
```
Comment (viktoria-rybenchuk) [inline taxi/views.py]: you dont't need use `all` before `order_by`

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by viktoria-rybenchuk — APPROVED** [review]
Good job!

**Pull request closed without merging**